### PR TITLE
Some refactoring related to the argument 'clusterinstance'

### PR DESCRIPTION
**Pull request by zeldinha**
With this refactor it should be easier to add future functionality that makes using this a lot more user friendly. Got rid of 'clusterinstance' and separated it into cluster and instance for clarity.

### CODE REVIEW

**Comment by solarkennedy** [comment]
I think this is good, and I'm happy to see "args" not be a function argument for more things.

File: paasta_tools/paasta_cli/cmds/mark_for_deployment.py [hunk 1]
```diff
@@ -109,4 +113,21 @@ def paasta_mark_for_deployment(args):
...
             instance=instance,
         )
+    return returncode
+
+
+def paasta_mark_for_deployment(args):
+    """Wrapping mark_for_deployment"""
+    cluster, instance = args.clusterinstance.split('.')
+    service = args.service
+    if service and service.startswith('services-'):
```
Comment (Rob-Johnson) [inline paasta_tools/paasta_cli/cmds/mark_for_deployment.py]: can you explain what this and the next line is doing?

Comment (solarkennedy) [inline paasta_tools/paasta_cli/cmds/mark_for_deployment.py]: This that thing where jenkins prepends "services-" in as the service name arg

Comment (Rob-Johnson) [inline paasta_tools/paasta_cli/cmds/mark_for_deployment.py]: ah ok, makes sense. could you add a comment to explain that? In the long term it'd be good to move this logic out of paasta and into jenkins (we could just sed the $upstream_job_name that Jenkins passes as an arg)

Comment (solarkennedy) [inline paasta_tools/paasta_cli/cmds/mark_for_deployment.py]: I'll make @zeldinha handle that in the next pass

File: paasta_tools/paasta_cli/cmds/mark_for_deployment.py [hunk 2]
```diff
@@ -64,42 +65,45 @@ def build_command(
     instancename is where you want to deploy. E.g. cluster1.canary indicates
     a Mesos cluster (cluster1) and an instance within that cluster (canary)
     """
-    cmd = 'git push -f %s %s:refs/heads/paasta-%s' % (
+    cmd = 'git push -f %s %s:refs/heads/paasta-%s.%s' % (
```
Comment (mrtyler) [inline paasta_tools/paasta_cli/cmds/mark_for_deployment.py]: Hard-coded . is a mistake.

**Pull request merged**